src/core/models.py

```diff
@@ -196,6 +196,16 @@ class EventInfo(models.Model):
         choices=REMOTE_POLICY_CHOICES
     )
 
+    FIRST_SPEAKER_CHOICES = (
+        (True, _('Yes')),
+        (False, _('No'))
+    )
+
+    first_time_speaker = models.BooleanField(
+        verbose_name=_('first time speaker'),
+        default=False,
+        choices=FIRST_SPEAKER_CHOICES)
+
```

Comment:
`EventInfo` is a general model inherited by many other models. I think these lines should be moved to a model in `proposals/models.py`.

Reply:
Thanks for your reminder. I revise it.